src/pages/onboardingModal/iamPolicy/instructions.tsx

```diff
@@ -41,28 +41,33 @@ const IamPolicyInstructions: React.SFC<Props> = ({ t, s3BucketName }) => {
           >
             {`{
     "Version": "2012-10-17",
-    "Statement": [{
-        "Sid": "VisualEditor0",
-        "Effect": "Allow",
-        "Action": [
-            "s3:Get*",
-            "s3:List*"
-        ],
-        "Resource": [
-            "arn:aws:s3:::${s3BucketName}",
-            "arn:aws:s3:::${s3BucketName}/*"
-        ]
-    },
-    {
-        "Sid": "VisualEditor1",
-        "Effect": "Allow",
-        "Action": [
-            "s3:ListAllMyBuckets",
-            "iam:ListAccountAliases",
-            "cur:DescribeReportDefinitions"
-        ],
-        "Resource": "*"
-    }]
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "s3:Get*",
+                "s3:List*"
+            ],
+            "Resource": [
+                "arn:aws:s3:::bucket_name",
```

Comment:
we want to display the s3 bucket name the user enter. can you change it to 
```
         "Resource": [
            "arn:aws:s3:::${s3BucketName}",
            "arn:aws:s3:::${s3BucketName}/*"
        ]
```

Reply:
Ok, I've added the s3 bucket name